tests/cli/auth/test_login.py

```diff
@@ -81,16 +88,25 @@ def testOstorlabAuthLoginCLI_whenValidLoginCredentialsAreProvidedWithoutOtp_prom
     mock_prompt.return_value = None
     runner = CliRunner()
     token_dict = {"token": "2fd7a589-64b4-442e-95aa-eb0d082aab63"}
-    requests_mock.post(
-        login_runner.TOKEN_ENDPOINT,
-        [
-            {
-                "json": {"non_field_errors": ['Must include "otp_token"']},
-                "status_code": 400,
-            },
-            {"json": token_dict, "status_code": 200},
-        ],
+
+    responses = [
+        {
+            "json": {"non_field_errors": ['Must include "otp_token"']},
+            "status_code": 400,
+        },
+        {"json": token_dict, "status_code": 200},
+    ]
+    response_idx = 0
+
+    def _request_callback(request, response_idx):
+        response = httpx.Response(**responses[response_idx])
+        response_idx += 1
+        return response
```

Comment:
Can't u achieve the same this by just adding multiple responses? same in `tests/runtimes/cloud/runtime_test.py:83` ?

Reply:
It doesn't work.